packages/gatsby-theme-project-portal/src/components/SiteSearch.stories.tsx

```diff
@@ -0,0 +1,195 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { userEvent, within } from "@storybook/testing-library"
+import * as AboutPageStories from "../layouts/AboutPageLayout.stories"
+import * as ContactPageStories from "../layouts/ContactPageLayout.stories"
+import * as ProjectDetailStories from "../components/ProjectDetail.stories"
+import SiteSearch from "./SiteSearch"
+import { createSearchIndex } from "../../utils/search"
+
+const meta: Meta<typeof SiteSearch> = {
+  component: SiteSearch,
+  tags: ["autodocs"],
+  includeStories: /^[A-Z]/,
+}
+
+type Story = StoryObj<typeof SiteSearch>
+export default meta
+
+export const manyProjects = {
+  nodes: [
+    { ...ProjectDetailStories.LongTitle.args },
+    { ...ProjectDetailStories.Minimum.args },
+    { ...ProjectDetailStories.NullValues.args },
+    { ...ProjectDetailStories.Open.args },
+    { ...ProjectDetailStories.Ongoing.args },
+    { ...ProjectDetailStories.Completed.args },
+  ],
+}
+
```

Comment:
Can we maybe move the `Primary` story to the top? It showcases all the standard features of this component, and would be the first thing people see when they open up the Storybook.